config.yaml

```
@@ -1,11 +1,24 @@
+### ----- Begin Common Configurables ----- ###
+
 defaults:
-  - model_configs:  Google-Gemma-2-27b-it #Meta-Llama-3.1-8B-Instruct, Meta-Llama-3.1-70B-Instruct, Phi-3-mini-128k-instruct, Phi-3-medium-128k-instruct, Google-Gemma-2-27b-it
+  # Meta: Meta-Llama-3.1-8B-Instruct, Meta-Llama-3.1-70B-Instruct
+  # Microsoft: Phi-3-mini-128k-instruct, Phi-3-medium-128k-instruct
+  # Alphabet: Google-Gemma-2-27b-it
+  - model_configs:  Google-Gemma-2-27b-it
 
 # how many hours to keep the API server running for (increasing this will dramatically increase wait time)
 time: 1
 
+# where the fail emails should go (otherwise Venia gets them!)
+# model_configs:
+#   mail_user: null@null.null
```

I'm not sure what to do about Venia's email being the default; I think della will attempt to figure out a user's email if it's left blank but that may not work if this gets ported to other places. So I put it here for people to fill in and didn't want to change the default to a fake email...